source/cleanup.c

```diff
@@ -48,6 +48,8 @@ static void *find_orphan() {
     if (IS_KERNEL_NON_SLAB_HEAP(child)) {
       /* object is allocated, therefore reachable */
       reachable[TOBJ_ADDR_TO_IDX(ktimer_base, current_timer)] = true;
+    } else if (child == (void *)TIMER2_NEXT_KERNEL) {
+      reachable[TOBJ_ADDR_TO_IDX(ktimer_base, current_timer)] = true;
```

Comment:
"reachable" means either allocated or freed and another free object is pointing to it. When we see that a node is pointing to 0xE281100C, this means that 0xE281100C is reachable, but the table is only for real kernel objects.